### ISSUE TITLE
Error in post-hoc analysis of Bayesian Repeated Measures ANOVA

### ISSUE THREAD

**Issue by Lumossssss** [jasp-stats/jasp-issues#981]
* JASP version: 0.13.1.0-64bit for Windows

* OS name and version: Win10
* Analysis: Two-way Bayesian repeated-measures ANOVA
* Bug description: When doing post-hoc tests, the following error showed up 
![image](https://user-images.githubusercontent.com/69629203/93686792-8bad6f80-fa6d-11ea-897d-7663fefc7fd8.png)

* Expected behaviour: results of post-hoc tests should appear 

#### Steps to reproduce:
1. Go to 'Bayesian Repeated Measures ANOVA'
2. Specify two Repeated Measures Factors "IT" (4 levels: IT1, IT2, IT3, IT4) & Region (4 levels: F, C, P, O)
3. Insert the variables respectively into 'Repeated Measures Cells'. It should already be in order.
4. Attempt running post-hoc tests and the error should appear. ``

I am attaching my data below.
[FN400_FCPO.zip](https://github.com/jasp-stats/jasp-issues/files/5250471/FN400_FCPO.zip)

**Comment by vandenman** [comment]
Thanks for reporting the issue! I've fixed this in the pull request linked above. Once that is merged into the codebase I will let you know. If you want you can, at that time, use a nightly version of JASP where the error does not occur.
